### ISSUE TITLE
[BUG] Register displayed everytime after restart of the app

### ISSUE THREAD

**Issue by fabrei** [nanu-c/axolotl#127]
### Description

If I start axolotl on my phone, every time I have to register my number. The first screen I get to see is the info screen of the app with the possibility to click on the next button to proceed with my registration. I am a bit confused, because my phone is already registered. If I wait for a couple of seconds, the screen changes to my contact list and a registration is not needed. It is a new effect after an update to version 0.8.0.

### Steps to Reproduce

1. Open axolotl first time.
2. Info screen of axolotl with a next button appears.
3. Be confused about to have to register.
4. Wait some seconds and the contact list appears.

**Expected behavior:** [What you expect to happen]
Direct loading of my contact list.

**Actual behavior:** [What actually happens]
Axolotl prompt me to register and I have to wait some seconds to see the contact list.

#### Versions

v.0.8.1

#### Device
Oneplus One Bacon

#### Link to Debug Log

[application-click-textsecure.nanuc_textsecure_0.8.1.log](https://github.com/nanu-c/axolotl/files/4496771/application-click-textsecure.nanuc_textsecure_0.8.1.log)

**Comment by Flaburgan** [comment]
Indeed, the registration screen is displayed below the layout, so if the phone lags, it can be seen. I have no idea why this had been built like that, a big refactor of the markup is needed IMO, but probably not right now. I'm adding that to v1.0 milestone @nanu-c what do you think?

**Comment by nanu-c** [comment]
Workflow is like that,
* app starts
* gui loads and doesn't know his state, maybe it's first start, asks for a websocket connection
* backend starts, connects with openwhisper, opens the websocket endpoint

Backend takes longer than the frontend so maybe add a loading screen?

**Comment by nanu-c** [comment]
But the original error here is, that the active session is broken, because the websocket connection to open whisper is ended. So @fabrei  do you have still this problem?
```
[36mINFO[0m[2020/04/18 13:56:40] [textsecure] No new messages                 
[37mDEBU[0m[2020/04/18 13:56:40] [textsecure] Websocket sending message       
[37mDEBU[0m[2020/04/18 13:57:41] [axolotl-ws] error: websocket: close 1006 (abnormal closure): unexpected EOF 
[31mERRO[0m[2020/04/18 13:57:41] [axolotl] error:  websocket: close 1006 (abnormal closure): unexpected EOF 
```

**Comment by fabrei** [comment]
Yeah the bug still exists for me..

**Comment by nanu-c** [comment]
Can you try to disable your adblocker if you use one?

**Comment by fabrei** [comment]
I'm usinguyen uadblock. first I closed signal client,disabled uadblock and start signal..but the problem still exists. the same after reboot of the phone. then I uninstalled uadblock,closed signal app and started again with either no effect. I dont have any other blocker.

**Comment by nanu-c** [comment]
Okey  do you still use version 0.8.1? If so please update if not, uninstall axolotl and install it again maybe?

**Comment by fabrei** [comment]
no I already used 0.8.8, but now I installed 0.8.9. After upgrade to 0.8.9, I uninstalled it via openstore and installed it again. But this did not change the issue.

**Comment by nanu-c** [comment]
really strange.

**Comment by nanu-c** [comment]
Ma

**Comment by nanu-c** [comment]
/home/phablet/.local/share/textsecure.nanuc/

**Comment by Flaburgan** [comment]
To delete the config file, one can use UTTT to remove the cache and everything.

**Comment by fabrei** [comment]
after opening the app, a register didn't work, because the app went into my chats after some seconds. but I did a deregister from the settings. after that I registered the app again and now it works; so the register hint is not displayed any more :) I

btw: a member of a chat group told me, that the person did not received a message from me, but the other members did. But now after a reregister it works again..
so this issue can be closed..thanks for your help :)
